frontend/src/app/app.component.ts

```diff
@@ -51,6 +53,35 @@ export class AppComponent implements OnInit {
     });
   }
 
+  getBrowserName() {
+    const  usrAgent = navigator.userAgent;
+    let tem;
+    let browserInfo = usrAgent.match (/(opera|chrome|safari|firefox|msie|trident(?=\/))\/?\s*(\d+)/i) || [];
+        if (/trident/i.test(browserInfo[1])) {
+
+        tem = /\brv[ :]+(\d+)/g.exec(usrAgent) || [];
+
+        return { name: 'IE', version: (tem[1] || '') };
+
+        }
+
+    if (browserInfo[1] === 'Chrome') {
+
+        tem = usrAgent.match (/\bOPR|Edge\/(\d+)/);
+
+        if (tem != null)   { return { name: 'Opera', version: tem[1] }; }
+
+        }
+
+      browserInfo = browserInfo[2] ? [browserInfo[1], browserInfo[2]] : [navigator.appName, navigator.appVersion, '-?'];
+
+    if (( tem = usrAgent.match(/version\/(\d+)/i)) != null) { browserInfo.splice(1, 1, tem[1]); }
```

Comment:
Hey Brian,

This is the only one of your suggesting I haven't squared away yet. I'm really not sure what this should be called. I'm having a difficult figuring out what exactly it is. I'll take a deeper look possibly tonight and tomorrow morning for sure, but I thought I'd see if we wanted to run with it since everything else has been fixed.

Reply:
@Dmac26 yah if I had to take a guess, maybe the intention was `term`, as in find the matching term? I think what you told me offline, `parsedBrowserInfo` is satisfactory.